specification/schemas/responses/MessageCreated.yaml

```diff
@@ -16,10 +16,7 @@ properties:
         additionalProperties: false
         properties:
           messageReference:
-            type: string
-            description: Your unique message reference.
-            format: uuid
-            example: da0b1495-c7cb-468c-9d81-07dee089d728
+            $ref: ../types/MessageReference.yaml
           messageStatus:
             description: The status of your message.
             $ref: ../enums/MessageCreatedStatus.yaml
```

Comment:
I believe we're missing the `links` section on this response

Reply:
added as a template and replaced on both response and request

Reply:
that'll do for now! when we add new resources we'll need to rename/refactor this bit to be more specific